src/main/java/com/notably/logic/LogicManager.java

```diff
@@ -24,21 +22,15 @@
 
     private final Model model;
     private final Storage storage;
-    private final AddressBookParser addressBookParser;
 
     public LogicManager(Model model, Storage storage) {
         this.model = model;
         this.storage = storage;
-        addressBookParser = new AddressBookParser();
     }
 
     @Override
     public void execute(String commandText) throws CommandException, ParseException {
         logger.info("----------------[USER COMMAND][" + commandText + "]");
```

Comment:
Shouldn't we add the `NotablyParser` usage here?

Reply:
Done!